GUI/coregui/Views/SimulationWidgets/PythonScriptWidget.cpp

```diff
@@ -104,19 +104,28 @@ void PythonScriptWidget::generatePythonScript(const MultiLayerItem *sampleItem,
     }
 }
 
+
+
 void PythonScriptWidget::onExportToFileButton()
 {
     QString dirname(m_outputDir);
     if (dirname.isEmpty())
         dirname = QStandardPaths::writableLocation(QStandardPaths::HomeLocation);
 
-    QString file_name = QFileDialog::getSaveFileName(
-        this, "Select file", dirname, "Python scipts (*.py)", 0, QFileDialog::DontResolveSymlinks);
+    QString filters("Python scripts (*.py);;All files (*.*)");
+    QString defaultFilter("Python scripts (*.py)");
+    QString defaultName = dirname + QString("/untitled.py");
```

Comment:
```suggestion
    QString defaultName = dirname + QString("/untitled");
```

It will keep dialog in line with all the other save dialogs of BornAgain.